File: CaseStudy_Module6_FontEnd_New/src/model/Shops.js
export class Shops{
    id
    name
    phone
    description
    email
    startTime
    endTime
    statusShops
    user
    city

    constructor(id, name, phone, description, email, startTime, endTime, statusShops, user, city) {
        this.id = id;
        this.name = name;
        this.phone = phone;
        this.description = description;
        this.email = email;
        this.startTime = startTime;
        this.endTime = endTime;
        this.statusShops = statusShops;
        this.user = user;
        this.city = city;
    }
}

File: CaseStudy_Module6_BackEnd/src/main/java/com/example/casemd6/model/Shops.java
package com.example.casemd6.model;

import lombok.Data;

import javax.persistence.*;
import java.sql.Time;
import java.util.Set;

@Entity
@Data
public class Shops {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String phone;
    private String description;
    private String email;
    private Time startTime;
    private Time endTime;
    private String statusShops = "2";
    @ManyToOne
    private User user;
    @ManyToOne
    private City city;
    @ManyToMany(fetch = FetchType.EAGER)
    private Set<Shippers> shippers;
}
